### PR TITLE
New guide: FreeRADIUS RADIUS Server

### PR DESCRIPTION
**Pull request by neelchauhan**
#### Author checklist (Completed by original Author)
- [X] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [X] If applicable, steps and instructions have been tested to work
- [X] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

File: docs/guides/security/freeradius_radius_server.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+You could also do an `MD5`-hashed or `Crypt`-hashed password. To generate an MD5-hashed password, run:
+
+```bash
+echo -n password | md5sum | awk '{print $1}'
+```
+
+Replace `password` with the desired password.
+
+You will get an hash of `5f4dcc3b5aa765d61d8327deb882cf99`. In `users` file, insert the following instead:
```
Comment (SergeCroise) [inline docs/guides/security/freeradius_radius_server.md:66]:
```suggestion
You will get a hash of `5f4dcc3b5aa765d61d8327deb882cf99`. In `users` file, insert the following instead:
```
see also:
https://www.merriam-webster.com/dictionary/hash#examples

File: docs/guides/security/freeradius_radius_server.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+
+## Prerequisites and assumptions
+
+The following are minimum requirements for using this procedure:
+
+* The ability to run commands as the root user or use `sudo` to elevate privileges
+* A RADIUS client, such as a router, switch or Wi-Fi access point
+
+## Installing Tailscale
```
Comment (SergeCroise) [inline docs/guides/security/freeradius_radius_server.md:24]: Tailscale ?

Comment (neelchauhan) [reply docs/guides/security/freeradius_radius_server.md]: Oops, fixed it now.

**Review by sspencerwire — APPROVED** [review]

**Comment by sspencerwire** [comment]
Thank you @neelchauhan and @SergeCroise !

**Pull request merged**